config/initializers/filter_parameter_logging.rb

```diff
@@ -3,4 +3,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += %i[
+  passw secret token _key crypt salt certificate otp ssn
```

Comment:
password is not needed anymore?

Reply:
No, at least according to the Rails PR that introduced that: `https://github.com/rails/rails/pull/40513#issuecomment-727598707`

Reply:
ah its used as a regex - good to know